src/visitors/bundler.js

```diff
@@ -177,7 +177,7 @@ class Bundler {
           node.components = {};
         }
 
-        if (ctx.result.some((e) => e.severity === messageLevels.ERROR)) {
+        if (!this.config.ignoreErrors && ctx.result.some((e) => e.severity === messageLevels.ERROR)) {
```

Comment:
Also, here we will create bundle even if there are unresolved `$ref`s, that is not exactly "validation error", rather a structural/syntax ones.

Is that intended behavior?

Reply:
Yes. I think we are OK with that. These refs will be in-place so next tool can catch the error.
I will need --force for docs preview. We will show there are errors but we will show docs preview too...